homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -20,36 +22,48 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         add_entities(devices)
 
 
-class MaxCubeShutter(BinarySensorEntity):
-    """Representation of a MAX! Cube Binary Sensor device."""
-
+class MaxCubeBinarySensorBase(BinarySensorEntity):
     def __init__(self, handler, device):
         """Initialize MAX! Cube BinarySensorEntity."""
-        room = handler.cube.room_by_id(device.room_id)
-        self._name = f"{room.name} {device.name}"
         self._cubehandle = handler
         self._device = device
+        self._room = handler.cube.room_by_id(device.room_id)
+        self._attr_entity_category = "diagnostic"
```

Comment:
Please use an impored constant: https://github.com/home-assistant/core/blob/0cc4b7219fdaf4afdd0dd72e3f3e1107d6096b72/homeassistant/const.py#L716

```suggestion
        self._attr_entity_category = ENTITY_CATEGORY_DIAGNOSTIC
```

Reply:
done 748b37d24174ef8a47821ee357494b56f081d29b